packages/stencil/src/generators/application/generator.spec.ts

```diff
@@ -99,13 +100,13 @@ describe('schematic:application', () => {
     ]);
   });
 
-  it('should configure lint target with flat config', async () => {
+  xit('should configure lint target with flat config', async () => {
```

Comment:
```suggestion
  it('should configure lint target with flat config', async () => {
```
The 'x' should not be there right?